bentoml/frameworks/gluon.py

```diff
@@ -0,0 +1,69 @@
+import os
+from bentoml.exceptions import MissingDependencyException
+from bentoml.service.artifacts import BentoServiceArtifact
+from bentoml.service.env import BentoServiceEnv
+
+
+class GluonModelArtifact(BentoServiceArtifact):
```

Comment:
Could you also add it to the docs frameworks page [here](https://github.com/bentoml/BentoML/blob/master/docs/source/frameworks.rst) and the main readme [here](https://github.com/bentoml/BentoML#ml-frameworks)?

Reply:
sure let me follow up on that with a PR.